NodeJS/src/RestClient.ts

```diff
@@ -1,5 +1,8 @@
 import { CookieJar } from "tough-cookie";
-import { log4js, STATION_JOB_PATH, STATION_GROUP_PATH, FETCH_SKILL_ID_PATH, EXTENSION_PATH, USER_PATH, REMOVE_AGENT_PATH, AGENT_JOB_PATH, FETCH_AGENT_BY_USERNAME_PATH, FETCH_AGENT_ID_PATH, DELETE_STATION_PATH, STATION_ONLY_PATH, SUBSCRIPTION_PATH, VERSION, SUB_ACCOUNT_KEY, lodash } from "./Constants";
+import { log4js, STATION_JOB_PATH, STATION_GROUP_PATH, FETCH_SKILL_ID_PATH, EXTENSION_PATH, USER_PATH, REMOVE_AGENT_PATH,
+    AGENT_JOB_PATH, FETCH_AGENT_BY_USERNAME_PATH, FETCH_AGENT_ID_PATH, DELETE_STATION_PATH, STATION_ONLY_PATH,
+    SUBSCRIPTION_PATH, VERSION, SUB_ACCOUNT_KEY, lodash ,FETACH_AUXCODE_BASE,FETCH_AUX_CODES,
```

Comment:
please correct the name of FETACH_AUXCODE_BASE in the entire file

Reply:
removed